sopel/module.py

```diff
@@ -496,6 +518,16 @@ def handle_example_bugs(bot, trigger, match):
     Under the hood, when Sopel collects the decorated handler it uses
     :meth:`sopel.bot.Sopel.register_url_callback` to register the handler.
 
+    .. versionchanged:: 7.0
+
+        The same function can be decorated many times with :func:`url` with
+        different URL patterns. This function will be then registered as URL
+        callback for each of these patterns.
```

Comment:
This is probably more verbose than it needs to be. All it really needs to say is that decorating multiple times will work as expected (add all the URL patterns). Specifying what happens with registration etc. is redundant.